policyengine_us/variables/gov/states/co/ccap/co_ccap_parent_fee.py

```diff
@@ -0,0 +1,98 @@
+from policyengine_us.model_api import *
+
+
+class co_ccap_parent_fee(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Colorado Child Care Assistance Program parent fee"
+    reference = (
+        "https://docs.google.com/spreadsheets/d/1EEc3z8Iwu_KRTlBtd2NssDDEx_FITqVq/edit#gid=468321263",
+        "https://docs.google.com/spreadsheets/d/1HtPiC2qxclzWfBa7LRo2Uohrg-RCBkyZ/edit#gid=582762342",
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=41",
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=62",
+    )
+    unit = USD
+    definition_period = MONTH
+    defined_for = "co_ccap_eligible"
+
+    def formula(tax_unit, period, parameters):
+        year = period.start.year
+        month = period.start.month
+        if month >= 10:
+            instant_str = f"{year}-10-01"
+        else:
+            instant_str = f"{year - 1}-10-01"
+        p = parameters(instant_str).gov.states.co.ccap
+        person = tax_unit.members
+        spm_unit = tax_unit.spm_unit
+        # Calculate base parent fee and add on parent fee.
+        agi = tax_unit("adjusted_gross_income", period.this_year)
+        hhs_fpg = spm_unit("snap_fpg", period) * MONTHS_IN_YEAR
+        num_child_age_eligible = tax_unit("co_ccap_eligible_children", period)
```

Comment:
same as below comment
```suggestion
        eligible_children = tax_unit("co_ccap_eligible_children", period)
```